system/system.proto

```diff
@@ -118,6 +118,8 @@ message RebootResponse {
 // A RebootMethod determines what should be done with a target when a Reboot is
 // requested.  Only the COLD method is required to be supported by all
 // targets.  Methods the target does not support should result in failure.
+// 
+// If a target does not support a method, `INVALID_ARGUEMENT` should be returned.
```

Comment:
Is this line redundant with the last line of the preceding paragraph?

Reply:
fixed